### PR TITLE
73132 Dashboard introduction

### PR DESCRIPTION
**Pull request by theo-ritense**

### CODE REVIEW

File: introduction/modules/dashboard-introduction.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+## Concepts
+
+* **Dashboard** A container for any number of widgets. 
+    Multiple dashboards can be configured, which will be shown to the user as tabs.
+* **Widget** Presents data from a datasource using a display-type to the user.
+* **Data-source** Retrieves data from any desired source. 
+    Every datasource can publish output features like `value`, `total` or `values`.
+* **Display-type** A display type is responsible for the visualisation of data. 
+    For example: a number, graph, table, or any custom.
```
Comment (Klaas-Ritense) [inline introduction/modules/dashboard-introduction.md:17]:
```suggestion
    For example: bar char, big number, meter, gauge, or any custom. 
```

File: introduction/modules/modules.md [hunk 2]
```diff
@@ -40,6 +40,11 @@ It also includes choice fields, a generic way to create lists of values that can
 selectboxes or radiobuttons). Some security features used to control access to any Valtimo endpoints are also part of 
 this module.
 
+### Dashboards
+
+Dashboards can provide the user with statistical information.
```
Comment (Klaas-Ritense) [inline introduction/modules/modules.md:45]:
```suggestion
Dashboards give users quick insight into the current state of the application. Usually by giving the user statistical information about cases and processes.
```

File: introduction/modules/dashboard-introduction.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+A dashboard contains a set of widgets that display data in a customized way. 
+These dashboards and widgets can be configured by an administrator accordingly.
+
+## Concepts
+
+* **Dashboard** A container for any number of widgets. 
+    Multiple dashboards can be configured, which will be shown to the user as tabs.
+* **Widget** Presents data from a datasource using a display-type to the user.
+* **Data-source** Retrieves data from any desired source. 
+    Every datasource can publish output features like `value`, `total` or `values`.
```
Comment (Klaas-Ritense) [inline introduction/modules/dashboard-introduction.md:15]:
```suggestion
* **Data-source** Every dashboard widget retrieves its data from a data-source. 
```

I don't think people need to know about response body fields: 'value', 'total', 'values'

Comment (theo-ritense) [reply introduction/modules/dashboard-introduction.md:15]: You're probably right that the introduction is not the right place for that information.

**Review by Klaas-Ritense — APPROVED** [review]

**Pull request merged**